data/json/monsterdrops/zombie_default.json

```diff
@@ -266,25 +266,30 @@
   {
     "type": "item_group",
     "subtype": "collection",
-    "//": "contains cloth (always), a wallet (30% chance), and items in pockets or a bag (50% chance)",
+    "//": "contains clothing (always), a wallet (30% chance), and items in pockets or a bag (80~% chance)",
     "id": "default_zombie_death_drops",
     "entries": [
+      { "group": "wallets", "damage": [ 1, 4 ], "prob": 30 },
       {
         "distribution": [
-          { "group": "default_zombie_clothes", "prob": 90 },
-          { "group": "default_zombie_clothes_christmas", "event": "christmas", "prob": 10 },
-          { "group": "professional_zombie_gear", "prob": 1 }
+          {
+            "collection": [
+              { "group": "default_zombie_clothes", "prob": 90 },
+              { "group": "default_zombie_clothes_christmas", "event": "christmas", "prob": 10 },
+              { "group": "default_zombie_items", "prob": 80 }
+            ],
+            "prob": 96
+          },
```

Comment:
```suggestion
          {
            "collection": [
              {
                "distribution": [
                  { "group": "default_zombie_clothes", "prob": 90 },
                  { "group": "default_zombie_clothes_christmas", "event": "christmas", "prob": 10 }
                ]
              },
              { "group": "default_zombie_items", "prob": 80 }
            ],
            "prob": 96
          },
```

I didn't test this beyond linting, but I think it should look something like this for them to definitely spawn with one set of clothing.

Reply:
it does look like there's a bug in the logic of `event` I am getting some sparsely clothed zombies applying this patch
![Screenshot from 2022-12-25 22-57-53](https://user-images.githubusercontent.com/58050969/209498140-51ba8925-8e56-4c0d-ab21-6e876de7e674.png)
![Screenshot from 2022-12-25 22-58-42](https://user-images.githubusercontent.com/58050969/209498142-4ba3f13b-9c4c-4d73-aae8-86b25e60931e.png)

![Screenshot from 2022-12-25 22-57-40](https://user-images.githubusercontent.com/58050969/209498139-177fcfb9-176f-400d-910d-f00b9a3a2dc4.png)
by the way I did look at the christmas itemgroups, just didn't look hard enough I guess, I was like they're just hats

Reply:
> it does look like there's a bug in the logic of `event` I am getting some sparsely clothed zombies applying this patch 

I'm not seeing anything obvious, except christmas goes from the 14th to the 25th in the code, so depending on your timezone it might be over already.

Not sure what the issue with my suggestion really is. But I think your version has a 10% chance to roll naked zombies, since `default_zombie_clothing` has a 90% chance in a collection. I was trying to amend that by wrapping the normal and christmas clothing in a distribution so you always get one of them.